core/src/main/java/org/fao/geonet/MergeUsersByUsernameDatabaseMigration.java

```diff
@@ -76,7 +76,8 @@ private void mergeUsers(ApplicationContext applicationContext, String duplicated
 
         mergeGroups(applicationContext, duplicatedUserList, userToKeep);
         transferMetadata(applicationContext, duplicatedUserList, userToKeep);
-        transferSavedSelections(applicationContext, duplicatedUserList, userToKeep);
+        deleteMetadataStatus(applicationContext, duplicatedUserList, userToKeep);
```

Comment:
Is this required? See already done in `transferMetadata`: https://github.com/geonetwork/core-geonetwork/blob/72c8018f1b7abb9503676315237ad91669c91cb1/core/src/main/java/org/fao/geonet/MergeUsersByUsernameDatabaseMigration.java#L148

Reply:
Closing the PR. Don't know why it was not working without this extra method in my case. It has been deployed and tested successfully on my side ... another JPA mystery....